charts/fluid/fluid/templates/csi/daemonset.yaml

```diff
@@ -63,10 +63,11 @@ spec:
             allowPrivilegeEscalation: true
           image: "{{ .Values.csi.plugins.image }}"
           command: ["/usr/local/bin/entrypoint.sh"]
-          args :
+          args:
             - "--nodeid=$(NODE_ID)"
             - "--endpoint=$(CSI_ENDPOINT)"
             - --v=5
+            - "--pprof-addr=:6060"
```

Comment:
Do we need an option to control it?

Reply:
an option to control enable/disable pprof or control port numer